File: sdk-python/src/agentvend_agent_sdk/completion_status.py
from enum import Enum


class CompletionStatus(str, Enum):
    """Completion status for usage async completion (sdk-api-spec §3.3)."""

    COMPLETED = "COMPLETED"
    FAILED = "FAILED"

    @property
    def api_value(self) -> str:
        return self.value

File: sdk-java/src/main/java/com/tollara/client/model/CompletionStatus.java
package com.tollara.client.model;

/**
 * Completion status sent to the usage service (see docs/sdk-api-spec.md §3.3).
 */
public enum CompletionStatus {
    COMPLETED("COMPLETED"),
    FAILED("FAILED");

    private final String apiValue;

    CompletionStatus(String apiValue) {
        this.apiValue = apiValue;
    }

    /** JSON body value for the {@code status} field. */
    public String getApiValue() {
        return apiValue;
    }
}
